lca-chimevc-stack/lambda_functions/chime_call_analytics_initialization/index.js

```diff
@@ -623,11 +626,11 @@ const handler = async function handler(event, context) {
         );
         return;
       }
-
-      callData = await getCallDataFromChimeEvents(event);
+      
+      callData = await getCallDataFromDdb(event.detail.callId);
       await writeCallEndEventToKds(kinesisClient, callData.callId);
       callData.callStreamingEndTime = new Date().toISOString();
-      await writeCallDataToDdb(callData);
+      await writeCallDataToDdb(callData, callData.callId);
```

Comment:
we are now writing the stale data from DDB (i.e. getCallDataFromDdb(event.detail.callId);) rather than re-generating the callData from getCallDataFromChimeEvents.